cobalt/browser/h5vcc_system/h5vcc_system_impl.cc

```diff
@@ -58,4 +79,19 @@ void H5vccSystemImpl::GetLimitAdTracking(GetLimitAdTrackingCallback callback) {
   std::move(callback).Run(limit_ad_tracking);
 }
 
+void H5vccSystemImpl::GetUserOnExitStrategy(
+    GetUserOnExitStrategyCallback callback) {
+  std::move(callback).Run(GetUserOnExitStrategyInternal());
+}
+
+void H5vccSystemImpl::ConcealOrStop(ConcealOrStopCallback callback) {
+  auto strategy = GetUserOnExitStrategyInternal();
+  if (strategy == h5vcc_system::mojom::UserOnExitStrategy::kConceal) {
+    SbSystemRequestConceal();
```

Comment:
@kaidokert please work with Esmael to route this correctly on Android

Reply:
I have a question that came up in a conversation with @andrewsavage1 before: I saw the call here to `SbSystemRequestConceal()` and immediately thought this was not for AndroidTV, is that so? If yes then I guess this code should be sandwiched in an `use_evergreen` preprocessor clause? (maybe this is done in a BUIL.gn file, if so then just do assert and make that evident with a `#error`)

Reply:
In general we only want to use Starboard on Android for media/player functionality